### DISCUSSION TITLE
Allow forgetting multiple failed jobs at once

### DISCUSSION THREAD

**Discussion by chrisdicarlo** [laravel/framework#43725] [Ideas]
I am looking to use the Artisan `queue:forget `command to forget multiple IDs at once, similar to the `queue:retry command`. Implementation-wise it doesn't seem too complicated but I'm unclear about how to actually begin putting together a PR to the framework and more importantly if it would even be considered for merging?

I have an implementation but I have never contributed directly to the framework before.

Any direction or insight is greatly appreciated!

Cheers,
Chris

**Comment by Jmstewart1127hlg** [comment]
Old post but you can run these to retry or forget jobs specified by job class. 

Forget:
```sh
for uuid in $(php artisan queue:failed | awk '/MyJobClass/ {print $3}'); do
     php artisan queue:forget "$uuid";
done
```

Retry:
```sh
php artisan queue:retry echo $(php artisan queue:failed | awk '/MyJobClass/' | awk '{print $3}')
```
